platform/vendor/tekton/catalog/main/task/trivy-scanner/0.1/trivy-scanner.yaml

```diff
@@ -32,6 +32,14 @@ spec:
       description: Image or Path to be scanned by trivy.
       type: string
   steps:
+    - name: get-latest-cve-findings
```

Comment:
This file will get overwritten next time the `vendorme` is run. Meaning this change will not persist. We would have to make this change upstream in the tekton catalog. Another thing we could do is separate this out from tekton catalog but we would have to maintain it going forward which is not ideal.